### PR TITLE
Extensions for Clouditor UI

### PR DESCRIPTION
**Pull request by anatheka**
This PR adds
- color field in catalog
- Cloud Service tags
- short_name to Catalog

### CODE REVIEW

File: api/orchestrator/orchestrator.proto [hunk 1]
```diff
@@ -654,6 +654,10 @@ message CloudService {
     (tagger.tags) = "gorm:\"many2many:metric_configurations\"",
     (validate.rules).repeated .items.message.required = true
   ];
+
+  // a tag list of key/value pairs
```
[the remark this answers is not in the document]
Comment (anatheka) [reply api/orchestrator/orchestrator.proto:658]: Everything you want like in Azure, e.g. key = owner and value = lebogg. Do you really think we need an example?

File: api/orchestrator/orchestrator.proto [hunk 2]
```diff
@@ -682,6 +686,11 @@ message Catalog {
       items : {string : {pattern : "^(|basic|substantial|high|low|medium)$"}}
     }
   ];
+
+  string color = 8;
```
[the remark this answers is not in the document]
Comment (anatheka) [reply api/orchestrator/orchestrator.proto:690]: Yes, @oxisto needs a color property for the UI :)

File: api/orchestrator/orchestrator.proto (hunk 1 above)
[the remark this answers is not in the document]
Comment (anatheka) [reply api/orchestrator/orchestrator.proto:658]: I've added an example and changed the to a list, it was only one element.

**Review by oxisto — APPROVED** [review]

File: api/orchestrator/orchestrator.proto (hunk 1 above)
[the remark this answers is not in the document]
Comment (lebogg) [reply api/orchestrator/orchestrator.proto]: Don't assume that only we read this. For us it is clear. But for someone reading the API (externals or newcomers) and see the Tags field may ask for what that is useful

**Pull request merged**